internal/events/aggregator.go

```diff
@@ -88,14 +94,14 @@ func newAggregator(ctx context.Context, di database.Plugin, sh definitions.Defin
 }
 
 func (ag *aggregator) start() {
-	go ag.offchainListener()
+	go ag.batchRewindListener()
```

Comment:
This rename is not strictly required. I started down a path of looking at using the rewind functionality, to leave messages pending between claim and verification. However, as that would block any other claims to the identity on the same topic, and have other problems, I abandoned the approach.

I left the rename, as I think the new more generic naming is better.